test/pylib/scylla_cluster.py

```diff
@@ -298,6 +304,7 @@ def __init__(self, mode: str, exe: str, vardir: str,
         self.resourcesdir = pathlib.Path.cwd() / "test/pylib/resources"
         self.resources_certificate_file = self.resourcesdir / "scylla.crt"
         self.resources_keyfile_file = self.resourcesdir / "scylla.key"
+        self.endpoint_snitch = endpoint_snitch or "GossipingPropertyFileSnitch" if property_file else None
```

Comment:
So this patch not only sets endpoint-snitch if property-file is provided, but also allows specifying some "custom" endpoint-snitch config?

Reply:
yes, and `test_snitch_change` actually uses that since it relied on the snitch to remain as default despite providing a property_file

Reply:
OK. The new argument for manager::server_add() looks quite a big hammer against inability to setup custom snitch nail :( Can't snitch name be passed via existing config or cmdline argument?

Reply:
Hmm, good point.  I think this is feasible.
I'll take a stab at it

Reply:
done in next version